CanvasCore/CanvasCore/Todo/Network/TodoAPI.swift

```diff
@@ -23,6 +23,6 @@ open class TodoAPI {
     }
 
     open class func ignoreTodo(_ session: Session, todo: Todo) throws -> URLRequest {
-        return try session.DELETE(todo.ignoreURL)
+        return try session.DELETE(URL(string: todo.ignoreURL)?.path ?? todo.ignoreURL)
```

Comment:
why was this wrong before?

Reply:
`session.DELETE` and friends want paths, not full URLs. They append the passed in path to the `baseURL`, even if what's passed in is a fully qualified url.. 

I'm terrified of messing with Session and ThreeLegit stuff. This seemed like the least risky change.

I did attempt to make `Todo.ignoreURL` a `@NSManaged URL` but I guess `URI` types in core data models are a newish thing and the Todo data model file is too old.. I could bump up the version number but again, I'm afraid of breaking stuff.

Reply:
What if we still save a string, but it only be the path, since that's what we need.